src/Import/include/FMI3/fmi3_import_variable.h

```diff
@@ -395,14 +395,11 @@ FMILIB_EXPORT fmi3_boolean_t fmi3_import_get_float64_variable_relative_quantity(
 FMILIB_EXPORT fmi3_boolean_t fmi3_import_get_float64_variable_unbounded(fmi3_import_float64_variable_t* v);
 
 /** \brief Get start value for the variable */
-FMILIB_EXPORT fmi3_string_t* fmi3_import_get_string_variable_start(fmi3_import_string_variable_t* v);
+FMILIB_EXPORT fmi3_string_t fmi3_import_get_string_variable_start(fmi3_import_string_variable_t* v);
```

Comment:
Was wrong. I think the array ticket was merged a bit early, so fixed what was obviously wrong.